modules/installation-configuration-parameters.adoc

```diff
@@ -1654,96 +1681,122 @@ In {product-title} 4.13, the following vSphere configuration parameters are depr
 The following table lists each deprecated vSphere configuration parameter:
 
 .Deprecated VMware vSphere cluster parameters
-[cols=".^2,.^3a,.^3a",options="header"]
+[cols=".^2a,.^4a,.^2a",options="header"]
 |====
 |Parameter|Description|Values
 
-|`platform.vsphere.apiVIP`
-|The virtual IP (VIP) address that you configured for control plane API access.
-a|An IP address, for example `128.0.0.1`.
+l|platform
+    vsphere
+      apiVIP
+a|The virtual IP (VIP) address that you configured for control plane API access.
 
-[NOTE]
-====
-In {product-title} 4.12 and later, the `apiVIP` configuration setting is deprecated. Instead, use a `List` format to enter a value in the `apiVIPs` configuration setting.
-====
+*Note:* In {product-title} 4.12 and later, the `apiVIP` configuration setting is deprecated. Instead, use a `List` format to enter a value in the `apiVIPs` configuration setting.
```

Comment:
```suggestion
*Note:* In {product-title} 4.12 and later, the `apiVIP` configuration setting is deprecated.
Instead, use a `List` format to enter a value in the `apiVIPs` configuration setting.
```

Reply:
Thanks. I moved the notes to a footnote section. Not sure why the last column still won't wrap the text even though I specified a shorted table width.